CHANGELOG.unreleased.md

```diff
@@ -40,3 +40,4 @@ Section headings should be at level 3 (e.g. `### Added`).
 
 - Fixed ValueError on Windows when running a W&B script from a different drive (@jacobromero in https://github.com/wandb/wandb/pull/9678)
 - Fix base_url setting was not provided to wandb.login (@jacobromero in https://github.com/wandb/wandb/pull/9703)
+- Fix `WANDB_CONFIG_DIR` was not used for determining system settings (@jacobromero in https://github.com/wandb/wandb/pull/9711)
```

Comment:
Check grammar. You could write "Fix usage of `WANDB_CONFIG_DIR` when determining system settings", but it's simpler to say "Correctly use `WANDB_CONFIG_DIR` when determining system settings"